src/prefect/orion/database/orm_models.py

```diff
@@ -265,34 +266,29 @@ def estimated_run_time(self):
         """Total run time is incremented in the database whenever a RUNNING
         state is exited. To give up-to-date estimates, we estimate incremental
         run time for any runs currently in a RUNNING state."""
-        if self.state and self.state_type == schemas.states.StateType.RUNNING:
-            return self.total_run_time + (pendulum.now() - self.state.timestamp)
+        if self.state_type and self.state_type == schemas.states.StateType.RUNNING:
+            return self.total_run_time + (pendulum.now() - self.state_timestamp)
         else:
             return self.total_run_time
 
     @estimated_run_time.expression
     def estimated_run_time(cls):
-        # use a correlated subquery to retrieve details from the state table
```

Comment:
This expression property was on my radar for that other performance issue involving loading the run state (flow_runs/filter).  This might improve the performance of that endpoint too!